src/index.js

```diff
@@ -75,10 +76,20 @@ function requireAllModels(modelsDir) {
         excludeDirs: /^__tests__$/,
         filter: (fileName) =>
           isJavascriptOrTypescriptFileName(fileName) && !isTestFileName(fileName),
+        map: (name, filePath) => {
+          lastFileLoadedPath = filePath;
+          return name;
+        },
```

Comment:
I can't find a way of testing this without adding loads of unrelated fixtures. Structure & behavior of `index.js` highly relies on `fs` & project structure, and injecting `forest-express-sequelize/mongoose` within `forest-express` does not seems to be a good idea.

`require-all` is also not used everywhere in our code (Like [here](https://github.com/ForestAdmin/forest-express/blob/42fffaf52541aaa5edc9df33fb0645ec389bedbd/src/integrations/index.js#L6) ), so a solution using `mock-fs` does not work without adding a lot of extra mock data.

Moreover, this function is not exposed, thus impossible to unit test properly.

Any idea ?